react/src/App.js

```diff
@@ -7,9 +7,9 @@ import {
   Navigate,
   useLocation,
 } from "react-router-dom";
-import Userfront, { SignupForm, LoginForm, PasswordResetForm } from "@userfront/toolkit/react";
+import Userfront, { SignupForm, LoginForm, PasswordResetForm, LogoutButton } from "@userfront/toolkit/react";
 
-Userfront.init("demo1234");
+Userfront.init("rbv549xn");
```

Comment:
Theres a lot of inconsistency in the examples for what what to do with tenantId.  

IMO each of the examples should push the user to the dashboard and use their legit tenantId. 1) that gives the user an idea of what is going on 2) provides us a way to gather data for "auth'd user in live/test mode"

Reply:
I agree but not all of these examples are built to be used with a tenantID other than the ones coded into them.
- AWS API Gateway -> No tenantID necessary, README points to JWT in dashboard
- FastAPI -> Points to JWT and workspaceId in dashboard
- Flutter -> Points to workspaceId in the dashboard
- NextJS -> Not built to function with anything other than demo (it mocks the API)
- PHP -> Relies on the old pre-built forms so only works for the tenantId in the code (their toolIds are hard coded)
- React -> Points to workspaceId in the dashboard
- React Native -> Points to workspaceId in the dashboard
- Vue -> Points to workspaceId in the dashboard